packages/queued-request-controller/package.json

```diff
@@ -30,6 +30,7 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
+    "@metamask/approval-controller": "^5.0.0",
```

Comment:
Hmm, why are we adding this dependency? This package seems to only be used in the tests (and in fact, we don't technically need it there, but that's another thing).

Reply:
The `AddApprovalRequest` type from approval controller is a member of the `MiddlewareAllowedActions` union, which is used the package export `createQueuedRequestMiddleware`. I think this makes it qualify as a dependency?

Reply:
Er, yes, I must have missed that. Thanks.